acceptance-tests/tests/workflows/secretsFlow.spec.ts

```diff
@@ -23,52 +37,39 @@ describe('Secrets Flow', () => {
     testState.cleanup();
   });
 
-  describe('hs secrets add', () => {
-    it('should create a new secret', async () => {
-      await testState.cli.executeWithTestConfig(
-        ['secrets', 'add', SECRET.name],
-        [SECRET.value, ENTER]
-      );
-    });
-  });
+  it('should create a new secret', async () => {
+    await testState.cli.executeWithTestConfig(
+      ['secrets', 'add', SECRET.name],
+      [SECRET.value, ENTER]
+    );
 
-  describe('hs secrets list', () => {
-    it('should list the secret', async () => {
-      await expect
-        .poll(() => testState.cli.executeWithTestConfig(['secrets', 'list']), {
-          interval: 1000,
-          timeout: 10000,
-        })
-        .toContain(SECRET.name);
-    });
+    await waitForSecretsListToContainSecret(testState);
```

Comment:
I moved the check to the end of this test because it was had to tell if this test was failing because it had no `assertion`, so as long as the `hs secret add` exited successful it passed.  This will now check that the secret was added before passing the test.